<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGConditionEditorWidget</class>
 <widget class="QWidget" name="GGConditionEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageEdit">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="cmbVar">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbOp"/>
       </item>
       <item>
        <widget class="QLineEdit" name="txtValue"/>
       </item>
       <item>
        <widget class="QToolButton" name="btnOk">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/ok</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnCancel">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/abort</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageDisplay">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblDisplay">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnEdit">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/edit</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
